met-web/src/App.tsx

```diff
@@ -205,7 +205,7 @@ const App = () => {
                     backgroundColor: 'white',
                     zIndex: ZIndex.footer,
                     position: 'relative',
-                    paddingTop: '5em',
+                    // paddingTop: '5em',
```

Comment:
Is there a reason this was left in as a comment?

Reply:
Not really; aside from wondering if it was still used for something